engine/app/components/citizens_advice_components/checkbox_single.erb

```diff
@@ -0,0 +1,17 @@
+<div class="cads-form-field<% if error? %> cads-form-field--has-error<% end %>">
+  <% if error? %><div class="cads-form-field__error-marker"></div><% end %>
+  <div class="cads-form-group cads-form-field__content cads-form-group--checkbox cads-checkbox-single">
+    <% if error? %>
+      <p class="cads-form-field__error-message" id="<%= error_id %>">
+        <%= error_message %>
+      </p>
+    <% end %>
+    <div class="cads-form-group__item">
+      <%= tag.input(class:"cads-form-group__input", **checkbox.attributes(name, error: error?)) %>
+      <label class="cads-form-group__label"
+             for="<%= label_id %>">
+        <%= checkbox.label %>
+      </label>
+    </div>
+  </div>
+</div>
```

Comment:
I must admit, I also tried to convert `form_group.html.haml` to erb as part of this PR, however I was having issues with the classes not appearing as a string What I had was `<fieldset class="<%= fieldset_classes %>">` which resulted in:
![Screenshot 2022-09-07 at 16 40 22](https://user-images.githubusercontent.com/94383569/188930638-aea79dfc-8d43-4170-afa2-1d11446fc66f.png)
I assume there must me an easy way to make those classes in erb so any suggestions are welcome and I'll pick them up in a new PR

Reply:
Cool to leave that for another PR, there's probably a couple ways:

1. Call join on the array: `<fieldset class="<%= fieldset_classes.join(" ") %>">`
2. Use a tag helper, which accepts an array: `tag.fieldset(classes: fieldset_classes)`
3. Rails 6.1 introduces a class_names helper which maybe in future we could pull in to make working with classes easier (https://github.com/rails/rails/pull/37918). We still need to support Rails 6.0 though until public-website is upgraded.